zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -537,6 +539,360 @@ mod tests {
+                    account INTEGER NOT NULL,
+                    diversifier BLOB NOT NULL,
+                    value INTEGER NOT NULL,
+                    rcm BLOB NOT NULL,
+                    nf BLOB NOT NULL UNIQUE,
+                    is_change INTEGER NOT NULL,
+                    memo BLOB,
+                    spent INTEGER,
+                    FOREIGN KEY (tx) REFERENCES transactions(id_tx),
+                    FOREIGN KEY (account) REFERENCES accounts(account),
+                    FOREIGN KEY (spent) REFERENCES transactions(id_tx),
+                    CONSTRAINT tx_output UNIQUE (tx, output_index)
+                )",
+                NO_PARAMS,
+            )?;
+            wdb.conn.execute(
+                "CREATE TABLE IF NOT EXISTS sapling_witnesses (
+                    id_witness INTEGER PRIMARY KEY,
+                    note INTEGER NOT NULL,
+                    block INTEGER NOT NULL,
+                    witness BLOB NOT NULL,
+                    FOREIGN KEY (note) REFERENCES received_notes(id_note),
+                    FOREIGN KEY (block) REFERENCES blocks(height),
+                    CONSTRAINT witness_height UNIQUE (note, block)
+                )",
+                NO_PARAMS,
+            )?;
+            wdb.conn.execute(
+                "CREATE TABLE IF NOT EXISTS sent_notes (
+                    id_note INTEGER PRIMARY KEY,
+                    tx INTEGER NOT NULL,
+                    output_pool INTEGER NOT NULL,
+                    output_index INTEGER NOT NULL,
+                    from_account INTEGER NOT NULL,
+                    address TEXT NOT NULL,
+                    value INTEGER NOT NULL,
+                    memo BLOB,
+                    FOREIGN KEY (tx) REFERENCES transactions(id_tx),
+                    FOREIGN KEY (from_account) REFERENCES accounts(account),
+                    CONSTRAINT tx_output UNIQUE (tx, output_pool, output_index)
+                )",
+                NO_PARAMS,
+            )?;
+            wdb.conn.execute(
+                "CREATE TABLE IF NOT EXISTS utxos (
+                    id_utxo INTEGER PRIMARY KEY,
+                    address TEXT NOT NULL,
+                    prevout_txid BLOB NOT NULL,
+                    prevout_idx INTEGER NOT NULL,
+                    script BLOB NOT NULL,
+                    value_zat INTEGER NOT NULL,
+                    height INTEGER NOT NULL,
+                    spent_in_tx INTEGER,
+                    FOREIGN KEY (spent_in_tx) REFERENCES transactions(id_tx),
+                    CONSTRAINT tx_outpoint UNIQUE (prevout_txid, prevout_idx)
+                )",
+                NO_PARAMS,
+            )?;
+            Ok(())
+        }
+
+        let data_file = NamedTempFile::new().unwrap();
+        let mut db_data = WalletDb::for_path(data_file.path(), tests::network()).unwrap();
+        init_main(&db_data).unwrap();
+        init_wallet_db(&mut db_data, Secret::new(vec![])).unwrap();
```

Comment:
If you make the change to handle UAs in the `address` column, then we can also add a test data case here like for the other two.